src/gen/fold.rs

```diff
@@ -3,18 +3,22 @@
 
 #![allow(unreachable_code, unused_variables)]
 #[cfg(any(feature = "full", feature = "derive"))]
+#[cfg_attr(syn_enable_doc_cfg, doc(cfg(any(feature = "full", feature = "derive"))))]
```

Comment:
Files in the `gen` folder are automatically generated by code in `/codegen`, and shouldn't be edited manually. I don't see any corresponding changes in the codegen crate for these changes, so I'm guessing they're applied manually. Please don't manually edit code in `gen/`.

Reply:
Yeah, I wasn't sure and originally didn't want to modify this folder at all. Unfortunately the sed-regex replaced that occurrences as well.